### PR TITLE
Feat/count votes

### LINKED ISSUE #55: Count the number of accounts that vote for a proxy
Count the number of accounts that vote for a proxy

image

For example

https://www.alohaeos.com/vote/proxy/brockpierce1

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?
- Resolves #55

### Steps to test

1. `make run`

### CODE REVIEW

File: hapi/src/mod.cjs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const fetch = (...args) =>
+  import('node-fetch').then(({ default: fetch }) => fetch(...args))
+
+module.exports = {
+  fetch
+}
```
Comment (leisterfrancisco) [inline hapi/src/mod.cjs:6]: @tetogomez please could you validate this? More on [this link](https://github.com/node-fetch/node-fetch#commonjs)

**Comment by xavier506** [comment]
wesome work leister , question why do we need hyperion for this solution?

**Comment by leisterfrancisco** [comment]
@tetogomez I'm not sure if I need the guest role to query a table directly from webapp, please let me know that information to add the `guest` or `anonymous` permission to the `proxy_votes` table.

File: hasura/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
[the remark this answers is not in the document]
Comment (xavier506) [reply hasura/LICENSE:1]: looks good

File: hapi/src/mod.cjs (hunk 1 above)
Comment (tetogomez) [reply hapi/src/mod.cjs:6]: nice!

**Review by tetogomez — APPROVED** [review]
LGTM

**Pull request merged**